tests/run/extmethods2.scala

```diff
@@ -14,4 +14,25 @@ object Test extends App {
   }
 
   test given TC()
-}
\ No newline at end of file
+
+  object A {
+    given ListOps[T](xs: List[T]) {
+      def second: T = xs.tail.head
```

Comment:
Needs more examples involving extension methods with type and term parameters, as well as neg tests when names used for type and term parameters clash with those used for the given clause.